YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -610,6 +618,11 @@ void Repository::ask_to_checkout_modified_files()
     repo_auto_sync_ = false;
 }
 
+bool Repository::ask_for_idb_tracking()
+{
+     return (ask_yn(true, "Should the IDB be tracked?") == ASKBTN_YES);
```

Comment:
nit: remove outermost parentheses

Reply:
fixed in 1d1e98c73c4b5c6eec04e07bac56ccb00b47b515